spec/support/page_objects/trainees/summary.rb

```diff
@@ -12,7 +12,7 @@ class Summary < PageObjects::Base
       section :training_details, PageObjects::Sections::TrainingDetails, ".training-details"
       section :degree_details, PageObjects::Sections::DegreeDetails, ".degree-details"
 
-      element :submit_for_trn_button, "input[name='commit']"
+      element :review_this_record_link, "a.govuk-button", text: "Review this record"
```

Comment:
This should probably target by class/id. If the text changes it'll break.

Reply:
👍 I've switched to using an ID